html/css/main.css

```diff
@@ -1260,6 +1260,22 @@ body {
         border-radius: 9px;
         opacity: 90%;
     }
+    .itemboxes-container {
+        height: 10.5rem;
+        min-width: 90rem;
+        max-width: 90rem;
```

Comment:
In this case sizes should be made in pixels instead. If somebody, for some reason, choose to change the root font size of resources, then it'll break this resource because of the use of `rem`.